Makefile.pre.in

```diff
@@ -1761,7 +1733,7 @@ altinstall: commoninstall
 
 commoninstall:  check-clean-src @FRAMEWORKALTINSTALLFIRST@ \
 		altbininstall libinstall inclinstall libainstall \
-		sharedinstall oldsharedinstall altmaninstall \
+		oldsharedinstall altmaninstall \
```

Comment:
Since what's old is new again, should we `%s/oldshared/shared/g`?

Reply:
good idea